collector/cpu_darwin.go

```diff
@@ -45,7 +46,7 @@ import (
 import "C"
 
 // ClocksPerSec default value. from time.h
-const ClocksPerSec = float64(128)
+const ClocksPerSec = C.CLK_TCK
```

Comment:
We want to keep the float64() cast to avoid integer math.

Reply:
Ah, ack.